jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenterTest.java

```diff
@@ -48,6 +50,9 @@
 import org.jbpm.workbench.ht.service.TaskService;
 import org.junit.Before;
 import org.junit.Test;
+import org.kie.server.controller.api.model.spec.Capability;
+import org.kie.server.controller.api.model.spec.ServerTemplate;
+import org.kie.workbench.common.workbench.client.error.DefaultWorkbenchErrorCallback;
```

Comment:
DefaultWorkbenchErrorCallback and ClientRuntimeError imports are not used, maybe they can be removed